src/lender/coordinator.sol

```diff
@@ -137,12 +158,16 @@ contract EpochCoordinator is Auth, Math, FixedPoint  {
         else revert();
     }
 
+    /// an epoch can be closed after minimum epoch time has passed
+    /// closeEpoch creates a snapshot of the current lender state
+    /// if all orders can be fulfilled epoch is executed otherwise
+    /// submission period is started
```

Comment:
```suggestion
    /// an epoch can be closed after a minimum epoch time has passed
    /// closeEpoch creates a snapshot of the current lender state
    /// if all orders can be fulfilled epoch is executed otherwise
    /// submission period starts
```